src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figures {
+    private int side;
+    private int upAndDown;
+
+    public Rectangle(String color, int side, int upAndDown) {
+        super(color);
+        this.side = side;
+        this.upAndDown = upAndDown;
+    }
+
+    @Override
+    public double area() {
+        return 2 * (side + upAndDown);
+    }

```

Comment:
The formula for the area of a rectangle is incorrect. The area of a rectangle is calculated by multiplying the length of the sides. It should be `side * upAndDown` instead of `2 * (side + upAndDown)`.